src/components/layout.js

```diff
@@ -26,7 +26,7 @@ const useStyles = makeStyles(theme => ({
       height: '100%',
     },
     content: {
-      background: `linear-gradient(rgba(251, 250, 251, 0.79), 65.97%, rgba(255, 255, 255, 0)), url('/images/diamondsSolidPink.svg'), center center`,
+      background: `linear-gradient(rgba(251, 250, 251, 0.79), 65.97%, rgba(255, 255, 255, 0)), url('/images/diamondsSolidOrange.png'), center center`,
```

Comment:
original was really just a png in an svg wrapper anyway, so I cut out the middleman.